docs/en/reference/query-builder.rst

```diff
@@ -80,7 +85,7 @@ Working with QueryBuilder
 High level API methods
 ^^^^^^^^^^^^^^^^^^^^^^
 
-To simplify even more the way you build a query in Doctrine, you can take
+The simplest way to build a dynamic query with the ``QueryBuilder`` is by taking
```

Comment:
```suggestion
The most straightforward way to build a dynamic query with the ``QueryBuilder`` is by taking
```

I try as much as possible to avoid the word "simple", because nothing we do is _actually_ simple, and it depends on the point of view 😄

Reply:
To be fair, "simplest" does not imply simple: the simplest way to build a spaceship is not simple 😆 but I see your point

Reply:
Still "simple" and "simplest" suggest an easy to understand method. I try to avoid words like "simple" or "then you ONLY need to".